backend/src/test/java/de/sambam/gamenighttracker/controller/UserControllerTest.java

```diff
@@ -35,46 +36,172 @@
 
     @BeforeEach
     public void setup() {
+        Player player1 = Player.builder().name("Mario").color("red").score(123)
```

Comment:
Maybe extract the creation of games for testing into helper methods, helps with readability.

Reply:
I save that for the last 2 days of the project phase ^^